crates/test-framework/src/conformance.rs

```diff
@@ -0,0 +1,699 @@
+use proc_macro2::{Ident, TokenStream};
+
+fn equalf64() -> TokenStream {
```

Comment:
Notice that this function does not need to be expanded all the time, it is just a normal function, that can be part of the test framework, and that the tests should call.

Everything that can belong in the test framework as "normal" (non quoted) Rust code should be. We should try to expand the minimum amount of Rust code to cut down compile-times.